TsRandomizer/Archipelago/SlotDataParser.cs

```diff
@@ -26,7 +26,7 @@ public static Requirement GetPyramidKeysGate(string pyramidKeysGate)
 		{
 			//TODO: remove when clients & server are update with correct value
 			if (pyramidKeysGate == "GateMilitaryGate")
-				return Requirement.GateMilitairyGate;
+				return Requirement.GateMilitaryGate;
```

Comment:
I realize now that I didn't see this TODO, but I'm a little confused by it

This first one is `if "GateMilitaryGate"` (correct spelling) return `return Requirement.GateMilitairyGate;` (typo),
but the others are reversed,
`if "GateLakeSirineLeft"` (typo) return `return Requirement.GateLakeSereneLeft;` (correct),

which format does this need to be to not break the server?

Reply:
Ohhh I think I got it.

Server was correct for Military, but this mapping was needed until the client Military was corrected (which it was here, resolving that portion of the todo)
Server was incorrect for Serene, and this if puts it at the correct name until the server is corrected